tests/unit/functions.tcl

```diff
@@ -35,7 +35,7 @@ start_server {tags {"scripting"}} {
 
     test {FUNCTION - Create a library with wrong name format} {
         catch {
-            r function load [get_function_code LUA {bad\0foramat} test {return 'hello1'}]
+            r function load [get_function_code LUA {bad\0format} test {return 'hello1'}]
```

Comment:
I'm not sure what the goals are here, but I'm hoping the `\0` is all that makes a `bad`+`format`. Notably, tests pass, so it's probably fine...